src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -249,6 +249,7 @@ public void CompleteIpo(Participant participant, string participantRowVersion, P
             CompletedBy = completedBy.Id;
             CompletedAtUtc = completedAtUtc;
             AddPreSaveDomainEvent(new IpoCompletedEvent(Plant, ObjectGuid));
+            AddPostSaveDomainEvent(new Events.PostSave.IpoCompletedEvent(Plant, ObjectGuid));
```

Comment:
InvitationTests.cs should have a test on this: See CompleteIpo_ShouldAddCompleteIpoEvent
Same goes for